src/sentry/static/sentry/app/data/forms/projectGeneralSettings.jsx

```diff
@@ -62,15 +92,25 @@ const formGroups = [
       },
       {
         name: 'resolveAge',
-        type: 'number',
-
+        type: 'range',
         min: 0,
-        max: 168,
-        step: 1,
+        max: 30,
+
+        allowedValues: RESOLVE_AGE_ALLOWED_VALUES,
         label: t('Auto Resolve'),
         help: t(
           "Automatically resolve an issue if it hasn't been seen for this amount of time"
         ),
+        formatLabel: val => {
+          val = parseInt(val, 10);
+          if (val === 0) {
+            return 'Disabled';
+          } else if (val > 23 && val % 24 === 0) {
+            val = val / 24;
+            return val + ' day' + (val != 1 ? 's' : '');
```

Comment:
use `!==`?

Reply:
Surprised we don't have a pluralize function lying around. You can extract this if you have time, otherwise we should probably add this at some later stage (and have it also handle translation not just add s 😃)

Reply:
There is `tn`